server/index.js

```diff
@@ -117,7 +117,8 @@ app.get('/api/home/:projectId', (req, res, next) => {
   }
   const sql = `
     select "projectId",
-           "projectName"
+           "projectName",
+           "userId"
```

Comment:
You probably should make sure to include the `userId` in your `where` clause so that people can only see their own projects?